src/main/resources/application.properties

```diff
@@ -51,11 +51,11 @@ phinms.service=MVPS
 phinms.sql=select * from ${phinms.table} where applicationStatus is NULL and processingStatus = 'queued'?dataSource=phinMsDataSource&onConsume=update ${phinms.table} set applicationStatus='completed' where recordId=:#recordId&onConsumeFailed=update message_inq set applicationStatus='failed' where recordId=:#recordId
 
 
-aims.bucketName=bucket
-aims.AccessKey=accessKey1
-aims.SecretAccessKey=verySecretKey1
-aims.S3Url=http://127.0.0.1:8000
+aims.bucketName=tradingpartners-east.sandbox.aimsplatform.com/AIMSPlatform
+aims.AccessKey=
+aims.SecretAccessKey=
+aims.S3Url=
 aims.SQSUrl=
 aims.SQSNotificationURL=
-
+aims.url=${aims.bucketName}?amazonS3Endpoint=${aims.S3Url}&accessKey=${aims.AccessKey}&secretKey=${aims.SecretAccessKey}&proxyHost=gatekeeper.mitre.org&proxyPort=80
```

Comment:
This set of application properties isn't going to work: can you pull over from the test properties?